CHANGELOG.md

```diff
@@ -1,7 +1,20 @@
 # [Unreleased](https://github.com/pybamm-team/PyBaMM/)
 
+## Features
+
+- Renamed "Terminal voltage [V]" to just "Voltage [V]". "Terminal voltage [V]" can still be used and will return the same value as "Voltage [V]".
+- Added "Anode potential [V]", which is the value of the surface potential difference (`phi_s - phi_e`) at the anode/separator interface, commonly controlled in fast-charging algorithms to avoid plating. Also added "Cathode potential [V]", which is the value of the surface potential difference at the cathode/separator interface. ([#2740](https://github.com/pybamm-team/PyBaMM/pull/2740))
```

Comment:
Shouldn't it be "Positive electrode" and "Negative electrode" so it is consistent with the other variables?

Reply:
The full name for this would be "Negative electrode surface potential difference at the separator interface". But I think in the fast-charging literature it's just called "Anode potential". Which do you think is better here?